js/friday.js

```diff
@@ -1,34 +1,34 @@
-function onLoad() {
-    let today = new Date().getDay();
-    let element = document.getElementById("friday");
-    if (today === 5) {
-      element.innerHTML = "YES";
-      let month = new Date().getMonth();
+(function IsItFridayYet() {
+  const date = new Date();
+  const theMonth = date.getMonth();
+  const theDay = date.getDay();
+  const element = document.getElementById("friday");
+  const spans = document.getElementsByClassName("fallingElement");
+  let season = "";
```

Comment:
why lets and consts?

A `var` can only be scoped into a function and is therefore visible anywhere. `let` and `const` are block-scoped and are not visible inside of a simple code block of `{}`

but the real difference here is that a `const` will not have a value change, and a `const` CANNOT be empty. We actually could have written season as `let season;` - my reason for keeping it an empty string is that it's not a bad call to ensure we are dealing with a type of string, by setting something as an empty string.

That said, it cannot be done with a `const`, and we get to implicitly tell the code reviewer that these const variables are intentional, will never change, and that the let IS expected to change.